### PR TITLE
Cleanup the showAsText() methods of data classes

### LINKED ISSUE #193: Bugs found in drafting an order
image

image

Instead, a message should be displayed telling the user that you need to add dishes before you can redeem points.

### PR DESCRIPTION
**Pull request by px1099**
- Merge the `showAsTextShowAll()` and `showAsTextHidePrivate()` into `showAsText()`
- Adjust the `showAsText()` command of ReadOnlyOrder.java for alignment
- Change the return message of `confirmdraft`
- Make the Earned point only shown in the success message of `confirmdraft`

Resolve #193

### CODE REVIEW

**Review by kangmingtay — APPROVED** [review]
yup looks good!

File: src/seedu/addressbook/common/Utils.java [hunk 1]
```diff
@@ -66,4 +66,29 @@ public static String formatCurrency(double input) {
         DecimalFormat df = new DecimalFormat("0.00");
         return df.format(input);
     }
+
+    /**
+     * Create blank space to position the next String at an exact distance compared to the start of the prefix String.
+     * @param prefix the String after which the blank space is inserted.
+     * @param distance the distance between the start of the two String that the blank space between them aims to fill.
```
Comment (SalsabilTasnia) [inline src/seedu/addressbook/common/Utils.java:73]: You may have some typo here 'distance the distance...'

File: src/seedu/addressbook/data/menu/Menu.java [hunk 2]
```diff
@@ -78,7 +78,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return this.name.fullName;
+        return getAsText();
```
Comment (SalsabilTasnia) [inline src/seedu/addressbook/data/menu/Menu.java:81]: Just wondering why you changed here. getAsText() returns everything while this.name.fullname returns name only right?

Comment (px1099) [reply src/seedu/addressbook/data/menu/Menu.java:81]: The reason is that toString() is used for displaying all details of the added menu
Furthermore, having a full getAsText() for toString() does not  create conflict to different name value creates the same toString()
And also the fact that there is a dedicated method for getting the name from the menu

File: src/seedu/addressbook/common/Utils.java (hunk 1 above)
Comment (px1099) [reply src/seedu/addressbook/common/Utils.java:73]: It is not a typo, as it is highlighted differently in the IDE. It looks like this:
**_@param distance_** the distance between the ...

**Pull request merged**